sdk/healthinsights/azure-healthinsights-cancerprofiling/pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.azure-sdk-build]
+pylint = false
+pyright = false
```

Comment:
Can you open a tracking issue to undo this once work on this package starts up again?

Reply:
will do